tests/unit/models/test_util.py

```diff
@@ -40,6 +45,27 @@ def test_exponential_counter__reset(self):
             counter.reset()
 
 
+class TestBoundedSet(UnitTest):
+    def test_contains(self):
+        bset = BoundedSet(max_items=10)
+        bset.add(1)
+        assert 1 in bset
+
+    def test_bound(self):
+        bset = BoundedSet(max_items=10)
+        [bset.add(i) for i in range(11)]
+        assert len(bset._set) == 10
+        assert 0 not in bset
+
+    def test_lru(self):
+        bset = BoundedSet(max_items=10)
+        [bset.add(i) for i in range(10)]
+        assert 0 in bset
+        bset.add(0)
+        bset.add(10)
+        assert 0 in bset
```

Comment:
I think we need to assert that `1` is no longer in the set - that's the least recently used element.

Reply:
I mean we can assert both.